apps/VARA HF/uninstall

```diff
@@ -2,12 +2,8 @@
 
 APPDIR="${HOME}"/.local/share/applications
 
-#Finding your default wineprefix directory
-if [ -z "${WINEPREFIX}" ]; then
-    PREFIXDIR="${HOME}"/.wine #if "WINEPREFIX=" envvar is unset or set to empty string, wineprefix location is "~/.wine"
-else
-    PREFIXDIR="${WINEPREFIX}" #if "WINEPREFIX=" envvar was previously set, wineprefix location is "$WINEPREFIX"
-fi
+#Finding the wineprefix where VARA.exe is installed
+PREFIXDIR=$(find ${HOME} -name "VARA.exe" | grep -P -o "(.*)(?=/drive_c/)")
```

Comment:
Beware that nothing is preventing this variable from containing multiple lines. It is up to you if you actually want to deal with that possibility though.